jac-uk/digital-platform
# 1667 targeted outreach report

This PR fixes the optional chaining on the function.

To test:
`npm run nodeScript targetedOutreachReport.js`

And check the eslint on the functions folder
`cd functions`
`npm run lint`
